src/components/Popups/PopupList.scss

```diff
@@ -14,4 +14,12 @@
     .ba-PopupList-prompt {
         padding: 10px;
     }
+
+    // Changing the collab list max-height to avoid possible overlap with next page in IE11
+    // The following media query is specific to IE10+
+    @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+        .ba-ItemList {
```

Comment:
Should we move this new style to the existing media query for IE11 to contain the workarounds to one place?

Reply:
That sounds like a good idea, however, the existing workaround is applied generally to `.ba-Popup` whereas this is a specific component style change

Reply:
Would it be worthwhile to centralize all the related rules and overrides in PopupBase?

```
// The following media query is specific to IE10+
@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
    .ba-Popup {
        z-index: 0; // Changing z-index causes page to flicker in IE11
    }
    
    .ba-ItemList {
        max-height: 140px; // Prevent list overflow onto subsequent pages in IE11
    }
}
```